test/controllers/visitation.controller.test.js

```diff
@@ -232,6 +232,9 @@ describe('Get all properties', () => {
             expect(res).to.have.status(200);
             expect(res.body.success).to.be.eql(true);
             expect(res.body).to.have.property('schedules');
+            expect(propId.equals(res.body.schedules[0].propertyId)).to.be.eql(true);
```

Comment:
Nice thinking with the test, however, I think this can be improved:
The test will be in 2 categories (but no demarcation is needed)
1. The first section tests that valid visitation data are returned.
2. The second section tests that a valid property information data are returned in propertyInfo.

```suggestion
           
            expect(res.body.schedules[0].propertyId).to.be.eq(propId.toString());
            expect(res.body.schedules[0].userId).to.be.eql(adminId.toString());
            expect(res.body.schedules[0].name).to.be.eql(booking.name);
            // ... add more booking tests here
            expect(res.body.schedules[0].propertyInfo[0].name).to.be.eql(demoProperty.name);
            // ... add more property tests here
```